docs/multihost-networking-tutorial.md

```diff
@@ -0,0 +1,254 @@
+
+Once several time, run the command:
+
+    user@master:~$ swarm-docker ps
+    CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
+    45b19d76d38e        debian              "bash"              6 seconds ago       Up 5 seconds                            bugs18/boring_mccarthy
+    53e87693606e        debian              "bash"              6 seconds ago       Up 5 seconds                            bugs16/amazing_colden
+    b18081f26a35        debian              "bash"              6 seconds ago       Up 4 seconds                            bugs17/small_newton
+    f582d4af4444        debian              "bash"              7 seconds ago       Up 4 seconds                            bugs18/naughty_banach
+    b3d689d749f9        debian              "bash"              7 seconds ago       Up 4 seconds                            bugs17/pensive_keller
+    f9e86f609ffa        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs16/pensive_cray
+    b53a46c01783        debian              "bash"              7 seconds ago       Up 4 seconds                            bugs18/reverent_ritchie
+    78896a73191b        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs17/gloomy_bell
+    a991d887a894        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs16/angry_swanson
+    a43122662e92        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs17/pensive_kowalevski
+    68d874bc19f9        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs16/modest_payne
+    e79b3307f6e6        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs18/stoic_wescoff
+    caac9466d86f        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs17/goofy_snyder
+    7748d01d34ee        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs16/fervent_einstein
+    99da2a91a925        debian              "bash"              7 seconds ago       Up 5 seconds                            bugs18/modest_goodall
+    cd308099faac        debian              "bash"              7 seconds ago       Up 6 seconds                            bugs19/furious_ritchie
+
+As shown, the containers are disseminated over bugs{16...19}.
+
+---
+## Multi-hosts network
+A network overlay is needed so all the containers can be "plugged in" into the same virtual LAN. To create this network overlay, execute:
+
+    user@master:~$ swarm-docker network create -d overlay net
+    user@master:~$ swarm-docker network ls|grep "net"
+    c96760503d06        net                 overlay
+
+**And voilà !**
```

Comment:
What do you mean? [_voilà_ expression](https://www.wordnik.com/words/voil%C3%A0).